utils/tim_delay.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * tim_delay.c
+ *
+ *  Created on: 19-Feb-2021
+ *      Author: Sonal Shrivastava
+ */
+
+#include <stdint.h>
+
+#include "drivers/mss_timer/mss_timer.h"
+#include "CMSIS/system_m2sxxx.h"
+#include "tim_delay.h"
+#include "led.h"
+
+
+// Delay counter
+//static volatile uint32_t DelayCounter;
+volatile uint32_t DelayCounter;
```

Comment:
Did the `static volatile` qualifier not work, or how come it's commented out?

Reply:
In one of the SO post, it mentioned usually one of the specifiers is to be used and not mix up by using both. Should I use both?

Reply:
Okay, I made it `static volatile` type.

Reply:
[https://stackoverflow.com/questions/46010403/static-volatile-vs-static-vs-volatile-in-c](url)
Link for your ref. I do not think it will create any problem.